workspaces/ballerina/ballerina-extension/e2e-test/e2e-authoring/scripts/run-steps.sh

```
@@ -0,0 +1,56 @@
+FILES="$(find "$STEPS_DIR" -maxdepth 1 -name '*.step.js' -type f | sort | while read -r file; do
+  n="$(basename "$file" | grep -o '^[0-9]*')"
+  if [ "$n" -ge "$FROM" ] 2>/dev/null && [ "$n" -le "$TO" ] 2>/dev/null; then
+    echo "$file"
+  fi
+done)"
+
+[ -n "$FILES" ] || { echo "no matching steps in $STEPS_DIR" >&2; exit 1; }
+
+echo "daemon '$NAME' on :$PORT" >&2
+echo "steps: $FILES" >&2
+OUTPUT="$(cat $FILES | "$EXEC")"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)